dev-client/src/components/home/MapSearch.tsx

```diff
@@ -82,24 +82,39 @@ export default function MapSearch({zoomTo, zoomToUser, toggleMapLayer}: Props) {
   }
 
   async function querySuggestions(queryText: string) {
-    if (queryText.length >= 2) {
-      try {
-        const {suggestions: newSuggestions} =
-          await makeSuggestionsApiCall(queryText);
-        setSuggestions(newSuggestions);
-      } catch (e: any) {
-        if (e.name !== 'AbortError') {
-          throw e;
-        }
+    const coordRegex =
+      /^([-+]?[1-8]?\d(?:\.\d+)?),\s*([-+]?180(?:\.0+)?|[-+]?((1[0-7]\d)|([1-9]?\d))(?:\.\d+)?)$/;
```

Comment:
question: what are you trying to match here? looks like a pair of decimal numbers or an integer or a decimal number? it's hard to follow. Is there a coord regex in one of the libraries we have that we can reuse?